### PR TITLE
Implement random adapter

### PR DESCRIPTION
**Pull request by NavyAdmiral**
For [#164472932](https://www.pivotaltracker.com/story/show/#164472932)

### CODE REVIEW

File: core/adapters/random_test.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+		{"resulting string parses to a big.Int 2", false},
+		{"resulting string parses to a big.Int 3", false},
+		{"resulting string parses to a big.Int 4", false},
+		{"resulting string parses to a big.Int 5", false},
+	}
+
+	for _, tt := range tests {
+		test := tt
+		t.Run(test.name, func(t *testing.T) {
+			t.Parallel()
```
Comment (dimroc) [inline core/adapters/random_test.go]: We've made it a standard to only place `t.Parallel` at the top level and not in subtests. Let's place this on line 13.

File: core/adapters/random_test.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+func TestRandom_Perform(t *testing.T) {
+	tests := []struct {
+		name    string
+		errored bool
+	}{
+		{"resulting string parses to a big.Int 1", false},
+		{"resulting string parses to a big.Int 2", false},
+		{"resulting string parses to a big.Int 3", false},
+		{"resulting string parses to a big.Int 4", false},
+		{"resulting string parses to a big.Int 5", false},
```
Comment (dimroc) [inline core/adapters/random_test.go]: Are these test cases just running the same test 5 times? I think we can just collapse it to one.

File: core/adapters/random_test.go [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+	for _, tt := range tests {
+		test := tt
+		t.Run(test.name, func(t *testing.T) {
+			t.Parallel()
+			input := models.RunResult{}
+			adapter := adapters.Random{}
+			result := adapter.Perform(input, nil)
+
+			if test.errored {
+				assert.Error(t, result.GetError())
```
Comment (dimroc) [inline core/adapters/random_test.go]: errored is always `false` so we can remove this.

**Comment by se3000** [comment]
Awesome, thanks @NavyAdmiral!

**Pull request merged**